lib/contentful_converter/nodes/hyperlink.rb

```diff
@@ -27,10 +27,6 @@ def options
         hyperlink_option
       end
 
-      def hyperlink_option
```

Comment:
It looks like we had two methods called "hyperlink_option", which meant this one was never being hit; removing it to clean up the code